packages/vsx-registry/src/browser/vsx-extension.tsx

```diff
@@ -351,13 +431,30 @@ export class VSXExtensionComponent extends AbstractVSXExtensionComponent {
 }
 
 export class VSXExtensionEditorComponent extends AbstractVSXExtensionComponent {
+
+    protected scrollContainer?: HTMLDivElement;
+    protected header?: HTMLDivElement;
+    protected body?: HTMLDivElement;
+
+    componentDidMount(): void {
```

Comment:
you don't need all of these

Reply:
Hi @akosyakov, the reason I have these additional elements is because it seems that the absence of a `readme` on line 485 could potentially cause the `body` and `scroll-container` to not be rendered. Additionally the methods `setContainerHeights` and `updateOnResize` manipulate all 3 of these elements, so I need a ref for each to ensure that they exist.

Reply:
Since I am keeping track of these in `VSXExtension`, I went ahead and removed these redundancies as well as some of the checks inside `componentDidMount`. The refs are now set using the setter methods passed in as props and to `VSXExtensionEditorComponent`, and `componentDidMount` will trigger the initial header/body sizing calculations if the refs exist